Makefile

```diff
@@ -14,38 +14,23 @@
 # `clean` run clean on esptool2, rboot, and rboot-sampleproject with inherited
 #				 configuration
 
-ESPTOOL2      ?= $(abspath esptool2/esptool2.exe)
-SDK_BASE      ?= C:/esp_iot_sdk_v1.1.2
-XTENSA_BINDIR ?= C:/xtensa-lx106-elf/bin

```

Comment:
It would be better to do something like this:
-include "Personal_Makefile" or something

If the include does not exist the - prevents a failure. And then in your case put the stuff in Personal_Makefile. And then I think it is better not to commit that Personal_Makefile since it is only useful for you and probably noone else.